index.js

```diff
@@ -275,6 +276,16 @@ export default class WorkspacesPlugin extends Plugin {
       }
     };
 
+    /**
+     * In workflows where publishing is handled on C.I.,
+     * and not be release-it, the environment will often require
+     * that the lockfile be updated -- this is usually done by
+     * running the install command for the package manager.
+     */
+    if (await findUp('pnpm-lock.yaml')) {
```

Comment:
Why do we need the find-up? I think we expect folks to run the commands in the root of the repo (e.g. the top level directory), and we have at least one other check for `pnpm-lock.yaml` that doesn't use a find-up...

Reply:
do we have a check that requires running from the root directory / root of a monorepo?

Reply:
done!